ols/utils/suid.py

```diff
@@ -0,0 +1,12 @@
+"""Session ID utility functions."""
+
+import uuid
+
+
+def get_suid() -> str:
+    """Generate a unique session ID (SUID) using UUID4.
+
+    Returns:
+        A unique session ID.
+    """
+    return str(uuid.uuid4().hex)
```

Comment:
Just to be sure, we don't mind that uuid is not "partitioned" by `-`?
![image](https://github.com/openshift/lightspeed-service/assets/37311499/8d9b5b2d-f739-4dd9-8eb3-12d1faa4ac29)

Reply:
we actually expect the 1st format

Reply:
UUID without hyphens is still technically a valid UUID, but its format deviates from the standard and widely recognized hyphenated notation.
So what is the reason that we expect it without hyphens?

Reply:
- You need to ask original author Vishna
- Do you expect this change to be part of refactoring?
- Where do we get the info, that we must use UUID? We just need some session ID, that's it. Can be generated by other means

Reply:
It is outside of this PR scope. We'll resolve this offline.